src/test/regress/expected/failure_1pc_copy_append.out

```diff
@@ -161,24 +161,21 @@ SELECT * FROM pg_dist_shard s, pg_dist_shard_placement p
  copy_test    |  100400 | t            | 0             | 3             |  100400 |          1 |        8192 | localhost |     9060 |         101
 (2 rows)
 
-SELECT citus.mitmproxy('conn.onQuery(query="^SELECT").kill()');
+SELECT citus.mitmproxy('conn.onQuery(query="SELECT|COPY").kill()');
  mitmproxy 
 -----------
  
 (1 row)
 
 SELECT count(1) FROM copy_test;
-WARNING:  connection error: localhost:9060
-DETAIL:  server closed the connection unexpectedly
-	This probably means the server terminated abnormally
-	before or while processing the request.
+WARNING:  could not consume data from worker node
```

Comment:
Note that the changes on `failure_test_name.out` files are actually setting the output files back to work fine with the existing executor since we had previously changed those to work OK with the adaptive executor.